geode-wan/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/ReplicateRegionCommandDUnitTest.java

```diff
@@ -409,6 +409,8 @@ public void testSuccessfulExecutionWithReplicatedRegionAndSerialSenderDueToRecei
         "Exception org.apache.geode.cache.client.ServerConnectivityException in sendBatch. Retrying"));
     ignoredExceptionsList
         .add(IgnoredException.addIgnoredException("DistributedSystemDisconnectedException"));
+    ignoredExceptionsList.add(IgnoredException.addIgnoredException(
+        "Exception when running replicate command: java.util.concurrent.ExecutionException: org.apache.geode.cache.EntryDestroyedException"));
```

Comment:
Wouldn't it be better to add Exception classes instead of the text of the exception (not only to this last change, but to the other exceptions ignored)?